pkg/skaffold/kubernetes/log.go

```diff
@@ -156,14 +151,31 @@ func (a *LogAggregator) streamContainerLogs(ctx context.Context, pod *v1.Pod, co
 	tr, tw := io.Pipe()
 	cmd := exec.CommandContext(ctx, "kubectl", "logs", sinceSeconds, "-f", pod.Name, "-c", container.Name, "--namespace", pod.Namespace)
```

Comment:
Could we pass in the start time to `LogAggregator.startTime` to be the start time of the *deploy*, rather than when the `LogAggregator` was created?